Makefile

```
@@ -172,3 +172,6 @@ storybook: ## Launch the storybook
 build-storybook: ## Build the storybook
 	@echo "Building storybook..."
 	@yarn build-storybook
+
+update-sandbox: ## Push the local version of the simple example to the sandbox repository
+	./update-demo.sh
```

```suggestion
	./update-sandbox.sh
```

(there is no `update-demo.sh`)